apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -545,12 +567,32 @@ export class ApiModelGenerator {
         startingNode: astDeclaration.declaration,
         nodesToCapture
       });
-
-      const docComment: tsdoc.DocComment | undefined = this._collector.fetchMetadata(astDeclaration).tsdocComment;
-      const releaseTag: ReleaseTag = this._collector.fetchMetadata(astDeclaration.astSymbol).releaseTag;
-
-      apiMethod = new ApiMethod({ name, docComment, releaseTag, isStatic, typeParameters, parameters, overloadIndex,
-        excerptTokens, returnTypeTokenRange });
+      const declarationMetadata: DeclarationMetadata = this._collector.fetchMetadata(astDeclaration);
+      const docComment: tsdoc.DocComment | undefined = declarationMetadata.tsdocComment;
+      const effectiveReleaseTag: ReleaseTag = declarationMetadata.effectiveReleaseTag;
+      if (effectiveReleaseTag === ReleaseTag.Internal || effectiveReleaseTag === ReleaseTag.Alpha) {
+        return; // trim out items marked as "@internal" or "@alpha"
+      }
+      // If we have lower-release function overloads, include that information
+      const symbolMetadata: SymbolMetadata = this._collector.fetchMetadata(astDeclaration.astSymbol);
+      const releaseTag: ReleaseTag = (
```

Comment:
same comment from above, here

Reply:
Right. The `Collector` cares about the `AstSymbol` relationships when reporting errors about inconsistent tags.  Wheres for all the subsequent processing, the declarations can be processed independently and we really shouldn't care whether they share an `AstSymbol` or not (with regards to release tags at least).